src/main/java/receiver/BatteryParameter.java

```diff
@@ -0,0 +1,38 @@
+package receiver;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BatteryParameter {
```

Comment:
This class just holds values. Can we encapsulate the data together with more functionality? So that changes to add a parameters etc. would get localized